src/Troubleshooting/index.md

```diff
@@ -11,3 +11,76 @@ This is due to a Roblox update causing an incompatibility with Wayland's handlin
 This specific problem can be the same as many other root causes, a common one is your graphics card not supporting Vulkan, or is missing the graphics libraries required for Wine; you can see [Lutris Docs: Installing Drivers](https://github.com/lutris/docs/blob/master/InstallingDrivers.md)
 
 This may also be your GPU not supporting Vulkan or not supporting modern Vulkan; Use the OpenGL renderer, or set the installed DXVK version to one which includes a legacy version of Vulkan that your GPU supports.
+
+#### "Make sure there are no stable wineservers running"
+
+This is because it failed to run the ESync Shared Memory File, so you will manually have to kill the `wineserver`.
+
+You need to end the `wineserver` process. Using System Monitor from many Desktop Environments, or TUI System Monitors like Btop.
+Or you can wait until the wineserver process will end.
+
+If you want to make your life easier, you can also type `vinegar kill` in the terminal.
+
+#### Roblox Studio stuck on Loading Screen
```

Comment:
```suggestion
#### Roblox Studio stuck on splashscreen
```
Self-explanatory. It's not an actual loading screen and we want to make sure readers don't confuse it with vinegar's loading screen.